eslint.config.ts

```diff
@@ -145,7 +145,7 @@ const config = tsEslint.config([
     languageOptions: {
       parserOptions: {
         projectService: {
-          allowDefaultProject: ['eslint.config.ts', 'knip.ts'],
+          allowDefaultProject: ['eslint.config.ts', 'knip.ts', 'node_package/tests/*.test.ts'],
```

Comment:
I'd still like
```suggestion
          allowDefaultProject: ['eslint.config.ts', 'knip.ts', 'node_package/tests/**/*.test.ts'],
```
just in case we add tests in subdirectories later, but the current is OK too.